### ISSUE TITLE
[Feature Request]: Move note of omitted Frequency Tables up

### ISSUE THREAD

**Issue by phish108** [jasp-stats/jasp-issues#2026]
### JASP Version

0.17.1 to 0.95.5

### Bug Description

The note for omitted frequency tables is hidden below the table where users do not expect it

![Screenshot](https://user-images.githubusercontent.com/866505/220076492-cbb254be-2676-4e84-bc12-d71933c3fcb6.png)

**Comment by phish108** [comment]
I assume that this behaviour is independent from the platform.

**Comment by phish108** [comment]
No, but will do tomorrow. but even 10 should be sufficient for two of my variables. 

However, I would expect that the distinct values count is per variable instead of for all variables combined. Therefore, i would expect to see one table for each variable.

**Comment by shun2wang** [comment]
@phish108 Can you give a sample file that reproduces this problem? You can zip a .jasp file and upload it here. thanks.

**Comment by phish108** [comment]
OK, I ran @tomtomme's suggestion. This does add the tables, but to me this feels counter intuitive. 

I expected a behaviour like with [forcats' `fct_lump()`](https://forcats.tidyverse.org/reference/fct_lump.html). The note under the tables is really hard to miss, because it has nothing to do with the table above, but looks like an annotation to it.

For @shun2wang I attach the file I used in the above screenshot.

[nominal_scales_sample.jasp.zip](https://github.com/jasp-stats/jasp-issues/files/10797655/nominal_scales_sample.jasp.zip)

**Comment by shun2wang** [comment]
@phish108  thanks for your sample! 

>I expected a behaviour like with [forcats' fct_lump()](https://forcats.tidyverse.org/reference/fct_lump.html). The note under the tables is really hard to miss, because it has nothing to do with the table above, but looks like an annotation to it.

I like this idea, as @vandenman @Kucharssim discussed [here](https://github.com/jasp-stats/jaspDescriptives/pull/14#issuecomment-737950039) , maybe lump uncommon factor together levels into "other"(in JASP table we can set another name to distinguish it from "other" character if it's a level/value in the variable)，and we just need to make it clear in QML that if it below a **distinct values** then will do this, which might be more clear than an annotation?

**Comment by phish108** [comment]
> > I expected a behaviour like with [forcats' fct_lump()](https://forcats.tidyverse.org/reference/fct_lump.html). The note under the tables is really hard to miss, because it has nothing to do with the table above, but looks like an annotation to it.
> 
> I like this idea, as @vandenman @Kucharssim discussed [here](https://github.com/jasp-stats/jaspDescriptives/pull/14#issuecomment-737950039) , maybe lump uncommon factor together levels into "other"(in JASP table we can set another name to distinguish it from "other" character if it's a level/value in the variable)，we just need to make it clear in QML that if it below a **distinct values** then will do this or mabye also a choose for **Never/none** to display all , which might be more clear than an annotation?

These are two things.

1. I definitely share @vandenman's concerns about table annotation. For presentation purposes, I would split off the annotation from the table and have headers for all variables. If variables are omitted for some reason, a statement should be placed there. As it stands now, an annotation right below a table would be the position where I would expect to see statements about values in the table, such as information about confidence intervals and alike. 
2. Not sure that I grasp what you mean with QML, exactly. I would expect a second check box  with "lump rare values as other" and having the level "other" configurable if the checkbox is on. I would expect such option as default in the spirit of @vandenman's remark regarding the preferability of presenting something even if it is incomplete or broken. This also is in line with R's default behaviour of displaying long data to the user.

**Comment by tomtomme** [comment]
This needs to be fixed still (tested with 0.18.3). Simple: 
![Screenshot_20240204_193423](https://github.com/jasp-stats/jasp-issues/assets/4428185/a5b9fbd8-33d6-47f6-b227-6019ab9cac81)

**Comment by phish108** [comment]
@tomtomme like so?

**Comment by tomtomme** [comment]
@phish108 
Actually no :D
My proposal above was: 
"[Bug] More prominent annotation for omitted frequency table needed"
or maybe even better
"[Bug] Misplaced annotation for omitted frequency table"

**Comment by tomtomme** [comment]
Still there with 0.19.2 beta flatpak
file to see the bug 
[nominal_scales_sample.jasp.zip](https://github.com/user-attachments/files/17492013/nominal_scales_sample.jasp.zip)
